assimilation_code/programs/readme.rst

```diff
@@ -0,0 +1,206 @@
+Programs
+========
+
+This list of programs is separated (imprecisely) into groups which have similar functionality.
+Within each group they are sorted (approximately) by the order
+in which they might be used andor by their usefulness.
+
+">>" marks the most commonly used and up to date programs.
```

Comment:
There seemed to be quite a bit of redundancy and historical baggage among the programs,
so I thought it would be helpful to point out the ones that people use most often.
The technique (>>) was not my first or second choice, so if you like the idea, but not the technique,
we can figure out some other way to note them.

Reply:
it seems like it's sufficient to move less frequently used programs to the end of each section.  if there are programs that are really not of interest, they could be collected at the very end of this doc in a section titled 'infrequently used/special cases'.  or at the end of each section there could be a sub-heading like that to separate out the least likely needed programs.  but i'd certainly list the most commonly used programs first, without the >>.

Reply:
Ok that makes sense.  I do not have a better idea than (>>) 🦖  🐸  🍩